src/js/base/select/SelectButton.tsx

```diff
@@ -1,17 +1,17 @@
-import * as RadixSelect from "@radix-ui/react-select";
+import { Trigger, Value } from "@radix-ui/react-select";
```

Comment:
I prefer that this is unchanged. It makes our code match the examples in the docs and makes it obvious what Radix component `Trigger` and `Value` are associated with.